src/config.ts

```diff
@@ -4,6 +4,7 @@ export default {
   COMMAND_PREFIX: '$',
   INTRO_CHANNEL: 'introductions',
   GENERAL_CHANNEL: 'general',
+  BOT_CHANNEL: '700340820162641962',
```

Comment:
Maybe this could be an env. var like `GENERAL_CHANNEL_ID` since in development we have different channel IDs? But at the same time, maybe this isn't the best solution because we'll start increasing the number of env. vars 😅.

@eddiejaoude and @garretcharp, do you guys think every channel ID we need in the bot should be configurable, or will this result in too many env. variables?

Reply:
You are completely right @BOLT04 ! It should be an envar!! 👍